apps/api/src/persistence/database/types/ArangoCollectionId.ts

```diff
@@ -1,3 +1,5 @@
-const arangoCollectionIds = ['terms', 'vocabulary_lists', 'tags'] as const;
+const arangoCollectionIDs = ['terms', 'vocabulary_lists', 'tags'] as const;
```

Comment:
Same as above, multiple sources of truth for entities?

Reply:
`arangoCollectionIDs` is the source of truth for the collection IDs. I have renamed
`arangoCollectionIDs` in `getArangoCollectionID` to `entityTypeToArangoCollectionID`.

```ts
const entityTypeToArangoCollectionID: {
    [k in EntityType]: ArangoCollectionID;
} = {
    term: 'terms',
    vocabularyList: 'vocabulary_lists',
    tag: 'tags',
};
```
Is not the source of truth for the arango collection IDs. It can't be; they are referenced in the type. **BUT** we may want to write a unit test that ensures that
- Each collection ID is used once and only once in building the entity types
- or at a minimum that each collection ID is actually used
There may be scenarios where multiple entities can go to the same collection, but I don't anticipate us doing things that way.

Reply:
Maybe we could talk about this tomorrow.  Just want to understand this better.